### ISSUE TITLE
Failed to load config \"standard\" to extend from. 

### ISSUE THREAD

**Issue by pikadun** [eslint/eslint#11951]
**Tell us about your environment**

* **ESLint Version:6.0.1**
* **Node Version:v10.15.3**
* **npm Version:6.4.1**

**What parser (default, Babel-ESLint, etc.) are you using?**
default
**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
{
    "extends": "standard",
    "rules": {
        "indent": [
            "error",
            2
        ],
        "semi": [
            "error",
            "always"
        ],
        "space-before-function-paren": [
            "error",
            "never"
        ],
        "no-template-curly-in-string": "off"
    }
}
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
var a = 1;
```

```bash
eslint --debug compile.js
```

**What did you expect to happen?**
It will work well

**What actually happened? Please include the actual, raw output from ESLint.**
```
D:\repos\RedisManager\script>eslint --debug compile.js
  eslint:cli CLI args: [ '--debug', 'compile.js' ] +0ms
  eslint:cli Running on files +8ms
  eslint:ignored-paths baseDir = "D:\\repos\\RedisManager\\script" +0ms
  eslint:ignored-paths addPatternRelativeToCwd:
  eslint:ignored-paths   original = "/node_modules/*"
  eslint:ignored-paths   cooked   = "/node_modules/*" +2ms
  eslint:ignored-paths addPatternRelativeToCwd:
  eslint:ignored-paths   original = "/bower_components/*"
  eslint:ignored-paths   cooked   = "/bower_components/*" +1ms
  eslint:ignored-paths addPatternRelativeToCwd:
  eslint:ignored-paths   original = ".*"
  eslint:ignored-paths   cooked   = ".*" +1ms
  eslint:ignored-paths addPatternRelativeToCwd:
  eslint:ignored-paths   original = "!../"
  eslint:ignored-paths   cooked   = "!../" +1ms
  eslint:ignored-paths Looking for ignore file in D:\repos\RedisManager\script +1ms
  eslint:ignored-paths Could not find ignore file in cwd +1ms
  eslint:ignored-paths baseDir = "D:\\repos\\RedisManager\\script" +1ms
  eslint:ignored-paths addPatternRelativeToCwd:
  eslint:ignored-paths   original = "/node_modules/*"
  eslint:ignored-paths   cooked   = "/node_modules/*" +1ms
  eslint:ignored-paths addPatternRelativeToCwd:
  eslint:ignored-paths   original = "/bower_components/*"
  eslint:ignored-paths   cooked   = "/bower_components/*" +1ms
  eslint:ignored-paths Looking for ignore file in D:\repos\RedisManager\script +0ms
  eslint:ignored-paths Could not find ignore file in cwd +3ms
  eslint:file-enumerator Start to iterate files: [ 'compile.js' ] +0ms
  eslint:file-enumerator File: D:\repos\RedisManager\script\compile.js +1ms
  eslint:cascading-config-array-factory Load config files for D:\repos\RedisManager\script. +0ms
  eslint:cascading-config-array-factory No cache found: D:\repos\RedisManager\script. +1ms
  eslint:config-array-factory Config file not found on D:\repos\RedisManager\script +0ms
  eslint:cascading-config-array-factory No cache found: D:\repos\RedisManager. +2ms
  eslint:config-array-factory Loading package.json config file: D:\repos\RedisManager\package.json +2ms
  eslint:config-array-factory Loading JSON config file: D:\repos\RedisManager\package.json +2ms
  eslint:config-array-factory Error reading package.json file: D:\repos\RedisManager\package.json +1ms
  eslint:config-array-factory Config file not found on D:\repos\RedisManager +1ms
  eslint:cascading-config-array-factory No cache found: D:\repos. +5ms
  eslint:config-array-factory Config file not found on D:\repos +1ms
  eslint:cascading-config-array-factory No cache found: D:\. +2ms
  eslint:config-array-factory Config file not found on D:\ +2ms
  eslint:cascading-config-array-factory Loading the config file of the home directory. +2ms
  eslint:config-array-factory Loading legacy config file: C:\Users\wg\.eslintrc +2ms
  eslint:config-array-factory Config file found: C:\Users\wg\.eslintrc +59ms
  eslint:config-array-factory Loading {extends:"standard"} relative to C:\Users\wg\.eslintrc +17ms

Oops! Something went wrong! :(

ESLint: 6.0.1.

ESLint couldn't find the config "standard" to extend from. Please check that the name of the config is correct.

The config "standard" was referenced from the config file in "C:\Users\wg\.eslintrc".

If you still have problems, please stop by https://gitter.im/eslint/eslint to chat with the team.
```

**Are you willing to submit a pull request to fix this bug?**
Will do it if I can

**Comment by mysticatea** [comment]
Thank you for your report.

Looks like a duplicate of #11914. Please track that issue.
